api/src/devcomp/domain/errors.js

```diff
@@ -1,15 +1,21 @@
 import { DomainError } from '../../shared/domain/errors.js';
 
-class UserNotAuthorizedToFindTrainings extends DomainError {
-  constructor(message = "Cet utilisateur n'est pas autorisé à récupérer les formations.") {
+class ModuleDoesNotExistError extends DomainError {
+  constructor(message = 'The module does not exist') {
     super(message);
   }
 }
 
-class ModuleDoesNotExistError extends DomainError {
-  constructor(message = "Le module id n'existe pas") {
+class PassageDoesNotExistsError extends DomainError {
```

Comment:
J'aurais pu le dire avant, mais je ne viens de m'en rendre compte que maintenant 😕 : « does not exist » sans « s »

```suggestion
class PassageDoesNotExistError extends DomainError {
```

Reply:
fixed 😅